Packs/ServiceNow/Integrations/ServiceNowv2/Layouts/layoutscontainer-ServiceNowTicket.json

```diff
@@ -0,0 +1,683 @@
+{
+    "id": "ServiceNow Ticket",
+    "group": "incident",
+    "name": "ServiceNow Ticket",
+    "description": "",
+    "version": -1,
+    "fromVersion": "6.0.0",
+    "detailsV2": {
```

Comment:
please add more space for the ticket info and comments
please convert all numbers (severity, urgency, etc..) to strings